lib/versions.ts

```diff
@@ -7,20 +7,12 @@ interface WasmCrate {
 
 export const versions = {
   wasmBindgen: "0.2.91",
-  wasmBindgenFutures: "0.4.41",
   jsSys: "0.3.68",
-  webSys: "0.3.68",
 };
 
 export function verifyVersions(crate: WasmCrate) {
   verifyVersion(crate, "wasm-bindgen", versions.wasmBindgen);
-  verifyVersionAllowNone(
-    crate,
-    "wasm-bindgen-futures",
-    versions.wasmBindgenFutures,
-  );
   verifyVersionAllowNone(crate, "js-sys", versions.jsSys);
-  verifyVersionAllowNone(crate, "web-sys", versions.webSys);
```

Comment:
I believe all of these need to be kept in sync in order to work well with wasm-bindgen?

Reply:
I think these all need to be using the same version of wasm-bindgen (ex. web-sys has a dep on wasm-bindgen https://crates.io/crates/web-sys/0.3.69/dependencies). Them getting out of sync causes problems... the wasm-bindgen-cli enforces these checks IIRC.

Reply:
Tests pass fine, and I wasn't able to find any occurrences of "web-sys" and "wasm-bingen-futures" anywhere besides the versions files. If my understanding is wrong, fair enough. I'll close the PR 🙂

Reply:
It might be fine to only enforce the wasm-bindgen version and ensure no duplicates in the cargo metadata: https://github.com/rustwasm/wasm-bindgen/pull/2913#pullrequestreview-986817593